File: server/src/main/java/gg/mineral/server/command/impl/StopCommand.java
package gg.mineral.server.command.impl;

import dev.zerite.craftlib.chat.type.ChatColor;
import gg.mineral.api.command.Command;
import gg.mineral.api.command.CommandExecutor;

public class StopCommand extends Command {

    public StopCommand() {
        super("", "stop");
    }

    @Override
    public void execute(CommandExecutor commandExecutor, String[] arguments) {
        commandExecutor.msg(ChatColor.RED + "Stopping server...");
        commandExecutor.getServer().shutdown();
    }

}

File: server/src/main/kotlin/gg/mineral/server/command/impl/StopCommand.kt
package gg.mineral.server.command.impl

import gg.mineral.api.command.Command
import gg.mineral.api.command.CommandExecutor
import net.md_5.bungee.api.ChatColor

class StopCommand : Command("stop", "") {
    override fun execute(commandExecutor: CommandExecutor, arguments: Array<String?>) {
        commandExecutor.msg(ChatColor.RED.toString() + "Stopping server...")
        commandExecutor.serverSnapshot.server.shutdown()
    }
}
